internal/plugin/connectors/http/generic/README.md

```diff
@@ -143,6 +143,26 @@ powerful transformation features.  You can use `printf` for formatting and
 you can compose functions using pipes `|`.  See the text template package docs linked
 above for detailed information on these and other features.
 
+### `oauth1`
+Like `headers` this is another key section. The `oauth1` section is used to generate
+an OAuth1 `Authorization` header. This will override any `Authorization` header that
+you have previously specified in your configuration.
```

Comment:
We should make this last line bold since it's important for the user to see

Reply:
We still need to add a panic for this scenario